### PR TITLE
Added ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+### Issue description
+
+
+
+
```
Comment (KrauseFx) [inline ISSUE_TEMPLATE.md]: This space should make it more obvious that you can fill in the actual issue here

**Comment by vpolouchkine** [comment]
Running locally vs. CI?

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+### Issue description
```
Comment (asfalcone) [inline ISSUE_TEMPLATE.md]: Do we want to word this in the "What did you do? What did you expect to happen? What actually happened?" way?

Comment (KrauseFx) [inline ISSUE_TEMPLATE.md]: That's what :apple: does, and I think it's sometimes really annoying, not sure how I feel about it

**Comment by mfurtak** [comment]
Before we decide to merge this, let's talk about whether we want to remove any redundant information from the CONTRIBUTING.md

**Comment by KrauseFx** [comment]
Agreed :+1:

**Comment by KrauseFx** [comment]
This is super interesting: https://github.com/jessesquires/JSQMessagesViewController/blob/develop/.github/ISSUE_TEMPLATE.md

**Comment by KrauseFx** [comment]
It's a little too much I think, however adding the checkboxes would be awesome. What do you think?

**Comment by KrauseFx** [comment]
I added a checklist, what do you think? Any other items

**Comment by asfalcone** [comment]
Checklist looks good! :white_check_mark:

**Pull request merged**